extraConfigSriov.py

```diff
@@ -86,6 +111,9 @@ def run(self, _) -> None:
         time.sleep(60)
         print(client.oc("wait mcp sriov --for condition=updated --timeout=50m"))
 
+        self.check_pci_realloc(client, "sriov")
```

Comment:
create a third function called: `ensure_pci_realloc` that `if need_pci_realloc(...)` then `enable_pci_realloc`

Reply:
Done.